pkg/snapstore/snapshot_test.go

```diff
@@ -76,6 +76,55 @@ var _ = Describe("Snapshot", func() {
 			})
 		})
 
+		Context("when provied with list of snapshots where few have the same timestamp", func() {
```

Comment:
Why not merge with https://github.com/gardener/etcd-backup-restore/pull/131/files#diff-cd87350645939a896c52eacbb73c2729R32.  We can handle it under same case.

Reply:
I felt it would be better to separate levels of the test cases where first one handles vanilla case while the second one handles cases with same timestamps. WDYT?

Reply:
Make sense 👍